### DISCUSSION TITLE
Ribbon Text box Set Focus Method

### DISCUSSION THREAD

**Discussion by DavidLanham** [Krypton-Suite/Standard-Toolkit#974] [Ideas]
I have been using the Krypton Ribbon controls for over 10 years and I love it.  But, there is one thing that I feel would be nice and that is a focus method for the text boxes on the Ribbon control.  Maybe I am doing it wrong, but I don't see a way to force the focus to the text box that I want.

thanks for keeping this going!

**Comment by Smurf-IV** [comment]
Sorry, I missed this..
Can you explain, 
Do you mean via tabbing, or click, or programatically (Ar all of the above ?)

**Reply by DavidLanham** [reply]
yeah a programmatically way like TextBox.Focus(); Like the Windows TextBox has.
